### DISCUSSION TITLE
执行检测模型微调时，开始训练报错

### DISCUSSION THREAD

**Discussion by ppppatchouli** [PaddlePaddle/PaddleOCR#12868] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- CentOS Linux release 8.5.2111
- Cuda compilation tools, release 12.0, V12.0.140
   Build cuda_12.0.r12.0/compiler.32267302_0
  cuDNN Version: 8.9
  python 3.8
- 版本号/Version：Paddle： develop
- PaddleOCR： 2.7.0.3
- 问题相关组件/Related components：PaddleOCR-release-2.7/tools/train.py
- 运行指令/Command Code：python PaddleOCR-release-2.7/tools/train.py -c ./ch_PP-OCRv3_det_studentLinux.yml
- 完整报错/Complete Error Message：
```bash
[2023/09/22 11:01:27] ppocr INFO: load pretrain successful from ./input0/ch_PP-OCRv3_det_distill_train/best_accuracy
[2023/09/22 11:01:27] ppocr INFO: During the training process, after the 0th iteration, an evaluation is run every 500 iterations
terminate called after throwing an instance of 'thrust::system::system_error'
  what():  radix_sort: failed on 2nd step: cudaErrorMissingConfiguration: __global__ function call is not configured

--------------------------------------
C++ Traceback (most recent call last):
--------------------------------------
0   paddle::pybind::ThrowExceptionToPython(std::__exception_ptr::exception_ptr)

----------------------
Error Message Summary:
----------------------
FatalError: `Process abort signal` is detected by the operating system.
  [TimeInfo: *** Aborted at 1695351689 (unix time) try "date -d @1695351689" if you are using GNU date ***]
  [SignalInfo: *** SIGABRT (@0x3ed000a6e6f) received by PID 683631 (TID 0x7f86a9424740) from PID 683631 ***]

Aborted (core dumped)
```

**我在此前已经运行了paddle.utils.run_check()来检测，其结果运行正常，并且使用预训练模型进行检测和识别并没有遇见问题。**
尝试使用断点追溯报错产生的时点，发现其在 loss_class = build_loss(config['Loss']) 这一步产生

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):
yes
请尽量不要包含图片在问题中/Please try to not include the image in the issue.

**Comment by SWHL** [comment]
如果问题仍然存在，请尝试最新main分支代码
